src/rdf4cpp/storage/identifier/NodeBackendID.hpp

```diff
@@ -163,7 +163,7 @@ static_assert(alignof(NodeBackendID) == alignof(uint64_t));
  * @return the LiteralType associated with that IRI
  */
 constexpr LiteralType iri_node_id_to_literal_type(NodeBackendID const id) noexcept {
-    assert(id.is_iri());
+    assert(id.null() || id.is_iri());
```

Comment:
I this function supposed to be called on the null node? If yes, what is it going to return?

Reply:
No, you are right. This was a mistake on my end